owl/EASE-OBJ.owl

```diff
@@ -1,14 +1,14 @@
 <?xml version="1.0"?>
 <rdf:RDF xmlns="http://www.ease-crc.org/ont/EASE-OBJ.owl#"
      xml:base="http://www.ease-crc.org/ont/EASE-OBJ.owl"
-     xmlns:owl="http://www.w3.org/2002/07/owl#"
      xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+     xmlns:owl="http://www.w3.org/2002/07/owl#"
      xmlns:xml="http://www.w3.org/XML/1998/namespace"
+     xmlns:swrl="http://www.w3.org/2003/11/swrl#"
      xmlns:xsd="http://www.w3.org/2001/XMLSchema#"
-     xmlns:EASE="http://www.ease-crc.org/ont/EASE.owl#"
+     xmlns:knowrob="http://knowrob.org/kb/knowrob.owl#"
```

Comment:
what is this namespace doing here?
SOMA ontology should not use knowrob namespace

Reply:
I dont know why they are added automatically from my protege. But I have removed them from the file.